samples/cloud-shell-oauth.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: workspace.che.eclipse.org/v1alpha1
+kind: Workspace
+metadata:
+  name: cloud-shell
+spec:
+  started: true
+  routingClass: openshift-oauth
+  devfile:
+    apiVersion: 0.0.1
+    metadata:
+      name: cloud-shell
+    components:
+      - alias: cloud-shell
+        type: cheEditor
+        id: eclipse/cloud-shell/nightly
+      - type: dockerimage
+        memoryLimit: 256Mi
+        alias: dev
+        image: 'quay.io/eclipse/che-sidecar-openshift-connector:0.1.2-2601509'
+        args: ["tail", "-f", "/dev/null"]
+        env:
```

Comment:
Does it make sense to add `USE_BEARER_TOKEN` here? Even if `false` by default it's a clue that it's configurable.

Reply:
Well, it's environment variable of cheEditor but not target container...
So, it makes sense to add it but - If I'm not mistaken - workspace-controller does not provision env vars for cheEditor...
The value of this env var should be provisioned by WorkspaceRouter Controller according to routing class... So, not sure we should promote it on devfile level.

Reply:
Merging this PR, for `USE_BEARER_TOKEN` will create a separate PR if agree that it makes sense.